webp/view/src/main/cpp/webp_decode.h

```diff
@@ -0,0 +1,76 @@
+/*
```

Comment:
I like to pardon this one, not like that tool to cut lines make code hard to read. sorry